tensorboard/components/tf_scalar_dashboard/tf-scalar-dashboard.html

```diff
@@ -100,165 +96,196 @@
         </div>
       </div>
       <div class="center">
-        <tf-panes-helper
-          categories="[[_categories]]"
-          data-type="[[dataType]]"
-          data-provider="[[dataProvider]]"
-          data-not-found="[[dataNotFound]]"
-          run2tag="[[run2tag]]"
-          selected-runs="[[_selectedRuns]]"
-          show-download-links="[[_showDownloadLinks]]"
-          download-link-url-function="[[scalarUrl]]"
-        >
-          <template>
-            <vz-line-chart
-              x-type="[[_xType]]"
-              color-scale="[[_runsColorScale]]"
-              smoothing-enabled="[[_smoothingEnabled]]"
-              smoothing-weight="[[_smoothingWeight]]"
-              tooltip-sorting-method="[[_tooltipSortingMethod]]"
-              ignore-y-outliers="[[_ignoreYOutliers]]"
-              ></vz-line-chart>
-            <paper-icon-button
-              class="log-button"
-              icon="line-weight"
-              on-tap="toggleLogScale"
-              title="Toggle y-axis log scale"
-              ></paper-icon-button>
-          </template>
-        </tf-panes-helper>
+        <template is="dom-if" if="[[dataNotFound]]">
+          <div class="no-data-warning">
+            <h3>No scalar data was found.</h3>
+            <p>Probable causes:</p>
+            <ul>
+              <li>You haven&rsquo;t written any scalar data to your event files.</li>
+              <li>TensorBoard can&rsquo;t find your event files.</li>
+            </ul>
+            <p>
+              If you&rsquo;re new to using TensorBoard, and want to find
+              out how to add data and set up your event files, check out
+              the
+              <a href="https://github.com/tensorflow/tensorflow/blob/master/tensorflow/tensorboard/README.md">README</a>
+              and perhaps the <a href="https://www.tensorflow.org/get_started/summaries_and_tensorboard">TensorBoard tutorial</a>.
+            </p>
+            <p>
+              If you think TensorBoard is configured properly, please
+              see
+              <a href="https://github.com/tensorflow/tensorboard/blob/master/README.md#my-tensorboard-isnt-showing-any-data-whats-wrong">the section of the README devoted to missing data problems</a>
+              and consider filing an issue on GitHub.
+            </p>
+          </div>
+        </template>
+        <!--
+          TODO(wchargin): Consider the extent to which this logic should
```

Comment:
+1 to the comment. Maybe we can encourage developers to use some components we provide within a dom-repeat like we do here.